assignment.js

```diff
@@ -66,6 +70,15 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const array1 = [57, 13, 4, 38]
+
+console.log(Math.max(...array1));
+
+const array2 = [3, 48, 29, 12, 18]
+
+console.log(Math.max(...array2));
+
```

Comment:
@thetripler Good Job here using the es6 spread operator syntax and utilizing the Javascript built-in library to return the largest value.
Take a look at this alternate approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L73)